src/main/kubernetes/artemis/deployments/usermanagement-service-deployment.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    field.cattle.io/creatorId: user-9jfkn
+  labels:
+    cattle.io/creator: norman
+    workload.user.cattle.io/workloadselector: deployment-artemis-usermanagement-service
+  name: usermanagement-service
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      workload.user.cattle.io/workloadselector: deployment-artemis-usermanagement-service
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        workload.user.cattle.io/workloadselector: deployment-artemis-usermanagement-service
+    spec:
+      containers:
+      - envFrom:
+        - configMapRef:
+            name: artemis-usermanagement-config
+            optional: false
+        - configMapRef:
+            name: jhipster-registry-config
+            optional: false
+        - secretRef:
+            name: artemis-secrets
+        image: <DockerId>/artemis-usermanagement-service
+        imagePullPolicy: Always
+        name: usermanagement-service
+        ports:
+        - containerPort: 8087
+          name: 8087tcp2
+          protocol: TCP
+        resources: {}
+        securityContext:
+          allowPrivilegeEscalation: false
+          privileged: false
+          readOnlyRootFilesystem: false
+          runAsNonRoot: false
+        stdin: true
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        tty: true
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
```

Comment:
Why are you specifying the scheduler?

Reply:
same as above - default config coming from the export, removed it